apps/courses/models.py

```diff
@@ -17,4 +17,4 @@ class Course(UUIDModel):
         blank=True,
     )
 
-    prerequisites = ManyToManyField("self", blank=True)
+    prerequisites = ManyToManyField("self", symmetrical=False, blank=True)
```

Comment:
i think we are missing the migration for this

Reply:
I runned makemigrations and it said nothing to change. It makes sense because adding this extra argument doesn't change the underlying table. It only help Django ORM in creating queries.

Reply:
perfect then 👍🏽